public/css/nitroxLauncher.css

```diff
@@ -285,73 +185,93 @@ h6 {
     visibility: hidden;
 }
 
+#titlebar #drag-region {
+    display: grid;
+    grid-template-columns: auto 138px;
+}
+
+/* controls buttons */
 #window-controls .button {
     grid-row: 1 / span 1;
     display: flex;
     justify-content: center;
     align-items: center;
     width: 100%;
     height: 100%;
+    cursor: pointer;
+    user-select: none;
+    position: relative;
+}
+
+#restore-button {
+    display: none !important;
 }
 
 #min-button {
     grid-column: 1;
 }
-
 #max-button, #restore-button {
     grid-column: 2;
 }
-
 #close-button {
     grid-column: 3;
 }
 
-@media (-webkit-device-pixel-ratio: 1.5), (device-pixel-ratio: 1.5),
-(-webkit-device-pixel-ratio: 2), (device-pixel-ratio: 2),
-(-webkit-device-pixel-ratio: 3), (device-pixel-ratio: 3) {
-    #window-controls .icon {
-        width: 10px;
-        height: 10px;
-    }
+/* controls buttons hover/active backgrounds */
+#window-controls .button::after {
+    content: "";
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    background-color: var(--control-other);
+    z-index: 0;
+    opacity: 0;
 }
-
-#window-controls {
-    -webkit-app-region: no-drag;
+#window-controls .button:hover::after {
+    opacity: .1;
 }
-
-#window-controls .button {
-    user-select: none;
+#window-controls .button:active::after {
+    opacity: .2;
 }
 
-#window-controls .button:hover {
-    background: var(--white-1);
+#close-button:hover::after {
+    background-color: var(--control-close-hover);
+    opacity: 1 !important;
 }
-
-#window-controls .button:active {
-    background: var(--white-2);
+#close-button:active::after {
+    background-color: var(--control-close-active);
+    opacity: 1 !important;
 }
-
-#close-button:hover {
-    background: #EB5757 !important;
+#close-button:active::before {
+    filter: invert(1);
 }
 
-#close-button:active {
-    background: #F1707A !important;
+/* controls buttons icons */
+#window-controls .button::before {
+    content: "";
+    background-color: #fff;
+    height: 100%;
+    width: 100%;
+    z-index: 10;
+    -webkit-mask-size: 23.1%;
```

Comment:
Here I would suggest we use `-webkit-mask-size: 10px;` instead, as the icons looked a bit blurry
![image](https://user-images.githubusercontent.com/62026364/79349731-e549c500-7f36-11ea-9fb1-246cc69fa6c4.png)